README.md

```diff
@@ -8,12 +8,12 @@
 
 ## 📌 Motive of the Project
 
-<b>Athavani or Memories</b> is a full-stack MERN tool built in order to save all your memories in a single place and rejoice them through the year.
+<b>Athavani or Memories</b> is a full-stack MERN tool built which is used to save all your memories in a single place so that you can enjoy them throughout the year.
 
 ## 💡 Project Background
 
-The Project is built with the help of Reactjs and Material-Ui at Frontend and backed by Nodejs and Expressjs at Backend using MongoDB as a Database. The user has the following
-ability
+The Project's Frontend is built with the help of Reactjs and Material-Ui  and it is backed by Nodejs. Its Backend is built using Expressjs using MongoDB as a Database. The user has the following
```

Comment:
Maybe this will be better
`The Project's Frontend is built with the help of Reactjs and Material-Ui. Its Backend is built using Nodejs and Expressjs using MongoDB as a Database. The user has the following`